File: src/main/webapp/js/Integration/ValueQuantity.js
function ValueQuantity(value, units, system, code) {
	this.value = value;
	this.units = units;
	this.system = system;
	this.code = code;
};

ValueQuantity.prototype.prettyPrint = function() {
	return "Value: " + this.value
	+ " Units: " + this.units
	+ " System: " + this.system
	+ " Code: " + this.code;
};

ValueQuantity.prototype.jsonPrint = function() {
	return { value: this.value,
	units: this.units,
	system: this.system,
	code: this.code };
};

module.exports = ValueQuantity;

File: src/main/java/Entities/ValueQuantity.java
package Entities;

import org.json.JSONObject;

public class ValueQuantity {

	private static final String CODE = "code";
	private static final String SYSTEM = "system";
	private static final String UNITS = "units";
	private static final String VALUE = "value";
	
	private JSONObject entity;
	
	public ValueQuantity(double value, String units, String system, String code) {
		entity = new JSONObject();
		entity.put(VALUE, value);
		entity.put(UNITS, units);
		entity.put(SYSTEM, system);
		entity.put(CODE, code);
	}
	public JSONObject getJSONObject() {
		return entity;
	}
}